src/config.ts

```diff
@@ -1,33 +1,71 @@
+    makeCache: Options.cacheWithLimits({
+        MessageManager: {
+            maxSize: Infinity,
+            sweepInterval: 300,
+            sweepFilter: LimitedCollection.filterByLifetime({
+                lifetime: 10800
+            })
+        },
+        ThreadManager: {
+            maxSize: Infinity,
+            sweepInterval: 300,
+            sweepFilter: LimitedCollection.filterByLifetime({
+                lifetime: 10800,
+                getComparisonTimestamp: e => e.archiveTimestamp!,
+                excludeFromSweep: e => !e.archived
+            })
+        }
+    }),
+    retryLimit: 3
+};
+export const shardsCount: number | "auto" = "auto";
+export const shardingMode: ShardingManagerMode = "worker";
+export const defaultPrefix = process.env.PREFIX?.replace(/"/g, "") ?? "!";
+export const owners: string[] = JSON.parse(process.env.OWNERS ?? "[]");
+export const embedColor = process.env.EMBED_COLOR?.toUpperCase() ?? "3CAAFF";
+export const devGuild = JSON.parse(process.env.DEV_GUILD ?? "[]");
+export const isProd = process.env.NODE_ENV === "production";
+export const isDev = !isProd;
+export const prefix = isDev ? "d!" : defaultPrefix;
+export const musicSelectionType = process.env.MUSIC_SELECTION_TYPE?.toLowerCase() as string || "message";
+export const is247Allowed = process.env.ENABLE_24_7_COMMAND?.toLowerCase() === "yes";
+export const stayInVCAfterFinished = process.env.STAY_IN_VC_AFTER_FINISHED?.toLowerCase() === "yes";
+export const yesEmoji = process.env.YES_EMOJI ?? "✅";
+export const noEmoji = process.env.NO_EMOJI ?? "❌";
+export const djRoleName = process.env.DJ_ROLE_NAME! || "DJ";
+export const lang = process.env.LOCALE?.toLowerCase() ?? "en";
+export const muteRoleName = process.env.MUTE_ROLE_NAME! || "Muted";
+
+i18n.configure({
+    defaultLocale: "en",
+    directory: join(__dirname, "..", "..", "lang"),
+    locales: [
+        "en",
+        "es"
+    ],
+    objectNotation: true
+});
+
+i18n.setLocale(lang);
 
-export const prefix = process.env.CONFIG_PREFIX?.replace(/"/g, "") ?? "!"; // Temporary workaround for https://github.com/docker/compose/issues/6951
-export const embedColor = process.env.CONFIG_EMBED_COLOR?.toUpperCase() ?? "7289DA";
-export const status = {
-    type: process.env.CONFIG_STATUS_TYPE?.toUpperCase() as ActivityType | null ?? "LISTENING",
-    activity: process.env.CONFIG_STATUS_ACTIVITY ?? "music on {guildsCount} guilds"
+export const presenceData: IpresenceData = {
+    activities: [
+        { name: i18n.__mf("activities.message1", { prefix }), type: "PLAYING" },
+        { name: i18n.__("activities.message2"), type: "LISTENING" },
+        { name: i18n.__("activities.message3"), type: "WATCHING" },
+        { name: i18n.__("activities.message4"), type: "PLAYING" },
+        { name: i18n.__("activities.message5"), type: "COMPETING" }
```

Comment:
Activity shouldn't be translated, the owner of bot can change this by anything they want

Reply:
Activities content will be moved to .env file

Reply:
I mean, it's better if we only translate static string instead of dynamic string